community: Add bio for Teodor Duțu (#3)
open-education-hub/open-education-hub-site: website/docs/community/people.md

# Project Team

## Teodor Duțu

![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)

Teodor Duțu is a contributor to the OpenEdu project (21-COP-0016).
He is developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/) and the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).

> Teodor is a Masters student at the University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.

Reviewer: ```suggestion
Teodor is a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.
```

Author: Are you sure? If it was just "University of Bucharest", there would definitely need to be a "the" before. Shouldn't it be "the POLITEHNICA University of Bucharest"? It sounds more natural to me like this.

Reviewer: `the POLITEHNICA University of Bucharest` sounds most natural to me, I'm not sure what the official name in english is for UPB. Maybe @razvand can give us the truth.

Reviewer 2: I tend to not use `the`.

Reviewer 2: But let's settle on something, and be consistent.

Reviewer: I updated my PR and I didn't use `the` there.

> He is also a teaching assistant at the University POLITEHNICA of Bucharest, being involved in the operating systems, data structures and hardware-software interface classes.

Reviewer: ```suggestion
He is also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
```

# Project Team

## Teodor Duțu

![Teodor Duțu](../../static/img/people/teodor_dutu.jpg)

I am a contributor to the OpenEdu project (21-COP-0016).
I am developing content for the [Operating Systems class](https://github.com/systems-cs-pub-ro/operating-systems-oer/), the [Security Summer School](https://github.com/security-summer-school/), as well as the [D Summer School](https://dlang-upb.github.io/D-Summer-School/).

I am a Masters student at University POLITEHNICA of Bucharest, Romania, enrolled in the Advanced Cybersecurity programme.

> I am also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.

Reviewer 2: ```suggestion
I am also part of teaching teams at University POLITEHNICA of Bucharest, being involved in the Operating Systems, the Data Structures and the Hardware-Software Interface classes.
```

> I am also a teaching assistant at the University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.

Reviewer: ```suggestion
I am also a teaching assistant at University POLITEHNICA of Bucharest, being involved in the Operating Systems, Data Structures and Hardware-Software Interface classes.
```

I am interested in compilers, programming language analysis, operating systems, security and embedded programming and passionate about open source and I am also a contributor to the [D programming language](https://dlang.org/).

> Apart from computer science, I like history, philisophy, literature, healthy cooking and running.

Reviewer 2: ```suggestion
Apart from computer science, I like history, philosophy, literature, healthy cooking and running.
```

I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.

> Apart from computer science, I really enjoy skiing, dancing and cooking.

Reviewer 2: Why is this line here as modified?

Reviewer 2: Is it because @lizababu didn't end her commit with a newline?

Author: Yes.
